### PR TITLE
Make VM_SUBMIT_ON_HOLD configurable using hiera data.

### PR DESCRIPTION
**Pull request by michael-os**
Add possibility to configure oned.conf VM_SUBMIT_ON_HOLD feature by means of the hiera variable one::oned::vm_submit_on_hold

### CODE REVIEW

File: templates/oned.conf.erb [hunk 1]
```diff
@@ -88,7 +88,7 @@ DB = [ backend = "mysql",
 
 VNC_BASE_PORT = 5900
 
-#VM_SUBMIT_ON_HOLD = "NO"
+VM_SUBMIT_ON_HOLD = "<%= scope.lookupvar('one::params::oned_vm_submit_on_hold') %>"
```
Comment (tuxmea) [inline templates/oned.conf.erb]: Is any value allowed here? Otherwise you need to validate the variable.

**Comment by michael-os** [comment]
Ok, I added validation and merged upstream changes. Ready for another round of review :)

**Comment by rwaffen** [comment]
looks good to me...

**Comment by tuxmea** [comment]
looks good to me, too

**Pull request merged**